src/workspaces/features/RunPipelineDialog/ParameterField.tsx

```diff
@@ -39,6 +42,18 @@ const ParameterField = (props: ParameterFieldProps) => {
       />
     );
   }
+
+  if (isConnectionParameter(parameter.type)) {
+    return (
+      <WorkspaceConnectionPicker
+        workspaceSlug={workspaceSlug || ""}
+        value={value ?? []}
```

Comment:
Why do we have an empty array as default / fallback value?

Reply:
It's for preventing the component to go from  uncontrolled to controlled. https://legacy.reactjs.org/docs/uncontrolled-components.html